code/game/objects/items/weapons/circuitboards/machinery/miscboards.dm

```diff
@@ -12,13 +12,47 @@
 /obj/item/weapon/circuitboard/sleeper
 	name = T_BOARD("sleeper")
 	desc = "The circuitboard for a sleeper."
+	build_path = "/obj/machinery/sleeper"
 	origin_tech = list(TECH_MAGNET = 2, TECH_ENGINEERING = 2)
-	board_type = "other" // change this to machine if you want it to be buildable
+	board_type = "machine"
 	req_components = list(
-							/obj/item/weapon/stock_parts/capacitor = 2,
-							/obj/item/weapon/stock_parts/scanning_module = 2,
-							/obj/item/weapon/stock_parts/console_screen = 1,
-							/obj/item/weapon/reagent_containers/glass/beaker/large = 1)
+							"/obj/item/weapon/stock_parts/capacitor" = 2,
+							"/obj/item/weapon/stock_parts/scanning_module" = 2,
+							"/obj/item/weapon/stock_parts/console_screen" = 1,
+							"/obj/item/weapon/reagent_containers/glass/beaker/large" = 1)
+
+
+/obj/item/weapon/circuitboard/bodyscanner
+	name = T_BOARD("bscannerm")
+	desc = "The circuitboard for a body scanner machine."
+	build_path = "/obj/machinery/bodyscanner"
+	origin_tech = list(TECH_MAGNET = 2, TECH_ENGINEERING = 2)
+	board_type = "machine"
+	req_components = list(
+							"/obj/item/weapon/stock_parts/capacitor" = 1,
+							"/obj/item/weapon/stock_parts/scanning_module" = 1,
+							"/obj/item/weapon/stock_parts/console_screen" = 1,
```

Comment:
Probably doesn't need a console screen since it doesn't have a built-in console.